### PR TITLE
feat(target_chains/ton): add ton deploy script

### PR DESCRIPTION
**Pull request by cctdaniel**
deployed to testnet at [`EQDwGkJmcj7MMmWAHmhldnY-lAKI6hcTQ2tAEcapmwCnztQU` ](https://testnet.tonscan.org/address/EQDwGkJmcj7MMmWAHmhldnY-lAKI6hcTQ2tAEcapmwCnztQU)

will create sdk for ton in a following PR and will update docs for ton testnet address

### CODE REVIEW

File: target_chains/ton/contracts/wrappers/Main.ts
Comment (ali-bahjati) [inline target_chains/ton/contracts/wrappers/Main.ts:1]: didn't we have these somewhere else?

**Review by ali-bahjati — COMMENTED** [review]
LGTM but I'm confused about similar codes interacting with the contract getting merged.

File: target_chains/ton/contracts/wrappers/Main.ts
Comment (cctdaniel) [reply target_chains/ton/contracts/wrappers/Main.ts:1]: hmm yeah will refactor them into a base wrapper

**Review by ali-bahjati — APPROVED** [review]
LGTM.

I am wondering whether having a "test" contract that exposes the same methods has any value at the moment (because you have more complexity on the TS side with BaseWrapper and two extensions of it) but I don't have full clarity to make a comment about it.

**Pull request merged**